docs/application/web/api/4.0/device_api/mobile/tizen/nfc.html

```diff
@@ -854,17 +794,125 @@ <h4>Attributes</h4>
 <div class="methods">
 <h4>Methods</h4>
 <dl>
+<dt class="deprecated method" id="NFCAdapter::setPowered">
+<a class="backward-compatibility-anchor" name="::NFC::NFCAdapter::setPowered"></a><code><b><span class="methodName">setPowered</span></b></code>
+</dt>
+<dd class="deprecated">
+<div class="brief">
+ Sets the power of an NFC adapter to either an on state or an off state.
+            </div>
+<p class="deprecated"><b>Deprecated.</b>
+ Deprecated since 2.3. Instead, let the user turn NFC on/off through the Settings application. See the <a href="../../../../../org.tizen.tutorials/html/web/tizen/network/nfc_tutorial_w.htm#Managing_NFC_Conn">Managing NFC</a> Tutorial.
```

Comment:
Please use the latest Tizen Developer Site link. This link is invalid now.

Reply:
It has been updated